spec/lib/hubspot/company_properties_spec.rb

```diff
@@ -82,15 +82,13 @@
 
     describe '.create!' do
       context 'with no valid parameters' do
-        cassette 'fail_to_create_property'
```

Comment:
This cassette does not exist. The `create!` functionality returns early with `nil`, instead of making an API call, when the post data is empty.